specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/examples/CreateOrUpdateASimpleGalleryImageVersion.json

```diff
@@ -16,24 +16,39 @@
               "regionalReplicaCount": 1,
               "encryption": {
```

Comment:
If you want this in the 2021-03-01 API release, make the changes in this branch : azure-rest-api-specs/specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01 at compute-2021-03 · Azure/azure-rest-api-specs (github.com)